typings/response.d.ts

```diff
@@ -0,0 +1,47 @@
+
+export interface DiffSummaryResult {
```

Comment:
For consistency with the other summary results, please can this be named `DiffResult`